content/broker/user-guides/developer-guides/events/_index.en.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Altinn 3 Broker Developer guides
+linktitle: Events
+description: How to get started subscribing to Events from Altinn 3 Broker, for developers
+tags: [Broker, guide, events]
+toc: true
+weight: 40
+---
+
+{{<children />}}
+
+{{% notice warning  %}}
+This section of the documentation is a work in progress, and as such currently makes extensive reference to external sources.
+{{% /notice %}}
+
+{{% notice warning  %}}
+Currently the Events for Broker are not ready for full-scale use, due to pending changes in Altinn Events and Authorization.
+This documents the expected scenario, but may be subject to change.
+{{% /notice %}}
+
+In order to use events/webhooks for a Broker resource, you need to setup a subscription for the given resource.
+This subscription is used to configure the endpoint where the events published by broker end up. [You can read more about how to setup an Events subscription in Altinn Events here](/events/subscribe-to-events/developer-guides/setup-subscription/).
+
+All Events published by Altinn Broker follow the same pattern:
+
+```json
+{
+ "id": "1faa107f-3c0a-4fa6-9fce-7cee8838e258",
+ "resource": "urn:altinn:resource:altinn-broker-test-resource-1",
+ "resourceinstance": "da4ceacc-ad44-4e54-99b6-b58e3c13c785",
+ "source": "https://platform.tt02.altinn.no/broker/api/v1/file",
+ "specversion": "1.0",
+ "type": "no.altinn.broker.Published",
+ "subject": "/party/50015641",
+ "alternativesubject": "/organisation/123456789",
+ "time": "2024-04-19T07:22:19.438039Z"
+}
+```
+
+## Event Subscription {#event-subcription}
+
+This subscription is used to configure the endpoint where the events published by broker end up. [You can read more about how to setup an Events subscription in Altinn Events here](/events/subscribe-to-events/developer-guides/setup-subscription/).
+
+You are required to set up the following filters:
+
+- sourceFilter
```

Comment:
Is SourceFilter used? And is it correct? We use filetransfer now.

Reply:
Currently the code sets "../file" in Source.
Will correct documentation to say "/filetransfer" and have added the change to Altinn/altinn-broker#402